rpa-home/modules/ROOT/pages/anypoint-migration-overview.adoc

```diff
@@ -0,0 +1,41 @@
+= Anypoint Platform Migration Overview
+
+Starting July 1st, RPA Manager integrates with Anypoint Platform to unify the login experience and provide support for external identities (IDP).
+
+If your organization acquired the MuleSoft Automation Bundle after the integration, your account is not affected by the migration because your organization is already in Anypoint Platform. 
+
+The migration process for existing RPA customers starts on July 1st and is carried out automatically in different waves. To prepare for the migration, ensure you follow the <<pre-migration, Pre-migration Steps>>.
```

Comment:
```suggestion
The automation migration process for existing RPA customers starts on July 1st and occurs in phases. To prepare for the migration, follow the <<pre-migration, pre-migration steps>>.
```

Reply:
(Not sure if that's correct. It might be nice to give some context like when the migration is expected to complete, what gets migrated first (region?), what to expect when migration is complete (an email? an in-app message?)